core/src/main/java/hudson/security/HttpSessionContextIntegrationFilter2.java

```diff
@@ -84,7 +84,13 @@ private boolean hasInvalidSessionSeed(Authentication authentication, HttpSession
             return false;
         }
 
-        User userFromSession = User.getById(authentication.getName(), false);
+        User userFromSession;
+        try {
+            userFromSession = User.getById(authentication.getName(), false);
+        } catch (IllegalStateException ise) {
```

Comment:
No-idiomatic abbreviations set a bad example to others. Common names for an exception are `e`, `x` and `ex`. Not a blocker.